site-src/concepts/gamma.md

```diff
@@ -0,0 +1,234 @@
+# The GAMMA initiative
```

Comment:
I really really hate the word "GAMMA". Maybe just a personal pet peave, but I would love to see this fade away.

IME its fine as "a name of a group of people". But the usage here is not great (not L1, I mean more like "How GAMMA works")

As a parallel, I feel like this is saying "How sig-network works" instead of "How `Service` works".

I would prefer something along the lines of "Gateway API for mesh", "Gateway for mesh", "Gateway for service mesh", "Gateway Service binding", ...

Reply:
Yeah, something to that. I think I've cleaned this up now, take another look?

Reply:
(Resolving optimistically, comment further if you want other changes!)